src/tools/cephfs/cephfs-shell

```diff
@@ -845,7 +845,7 @@ sub-directories, files')
         """
         for path in args.paths:
             if path.count('*') > 0:
-                files.extend([i for i in get_all_possible_paths(
+                args.paths.extend([i for i in get_all_possible_paths(
```

Comment:
Sory for introducing this bug. 

It's best not to modify args.paths directly. Copy the list into a local variable and modify that instead.

Reply:
I have updated the patch.